daml-foundations/daml-ghc/tests/GenericTemplates.daml

```diff
@@ -10,17 +10,18 @@ module GenericTemplates where
 import Prelude hiding (Template (..), Choice (..), create, fetch, exercise)
 
 class Template t where
+  -- TODO(MH): Decide if we actually want to provide `signatory` and
+  -- `observer` to DAML devs. If not, we can remove them here since they are
+  -- not required for template desugaring/resugaring anymore.
```

Comment:
I think we do - they seem like they could be useful - especially once you have generic templates!